Backport43 - Capping single-serevr support to 10 000 minions. (#1964)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/hardware-requirements.adoc

endif::[]

[IMPORTANT]
====

> {productname} performance depends on a number of factors, for example, underlying resources, network bandwidth, latency between clients and server etc.

Reviewer: ```suggestion
{productname} performance depends on  hardware resources, network bandwidth, latency between clients and server, etc.
```
